demo/test-fpga-region.yaml

```diff
@@ -13,7 +13,7 @@ spec:
           [IPC_LOCK]
     resources:
       limits:
-        fpga.intel.com/arria10.dcp1.2-nlb3: 1
+        fpga.intel.com/arria10.dcp1.1-nlb3: 1
         cpu: 1
```

Comment:
Can you remind me what's the reason of using old dcp version? Would it be better to flash hardware with the latest dcp release?

Reply:
Sure, I will flash to the latest DCP. However, I will keep 1.1 to make sure all test pass for now. As soon as I flash all three nodes to 1.2, I will then make the change back to 1.2